gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -509,14 +511,19 @@ public void handleLaunchFlowEvent(DagActionStore.DagAction launchAction) {
       // Upon handling the action, delete it so on leadership change this is not duplicated
       this.dagActionStore.get().deleteDagAction(launchAction);
     } catch (URISyntaxException e) {
-      log.warn("Could not create URI object for flowId {} due to exception {}", flowId, e.getMessage());
+      log.warn(String.format("Could not create URI object for flowId %s due to exception", flowId), e.fillInStackTrace());
```

Comment:
don't we want the true stack trace of the exception?  I would just pass this as `e` (no method invocation)